core/insertion_marker_manager.ts

```diff
@@ -611,12 +611,22 @@ export class InsertionMarkerManager {
     insertionMarker.queueRender();
     renderManagement.triggerQueuedRenders();
 
-    // Position so that the existing block doesn't move.
-    insertionMarker.positionNearConnection(imConn, closest);
     // Connect() also renders the insertion marker.
     imConn.connect(closest);
 
+    const originalOffsetToTarget = {
+      x: closest.x - imConn.x,
+      y: closest.y - imConn.y,
+    };
+    const originalOffsetInBlock = imConn.getOffsetInBlock().clone();
+    const imConnConst = imConn;
```

Comment:
nit: You don't need to assign this to another variable. You can just pass it directly.

Reply:
Hum, no.
These values are modified during the render.
That's why I get their "original" value in constants before the render.

Reply:
Oh, I see.